.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Hi, thank you for your contribution to Mantl! Before we can accept any code into
+master, we need it to meet the following criteria. If there are any you can't
+satisfy yourself, go ahead and open the issue anyway and we'll help you test.
+Feel free to delete this message once you're done. Thanks again!
+
+- [ ] Installs cleanly on a fresh build of most recent master branch
+- [ ] Upgrades cleanly from the most recent release
+- [ ] Updates documentation relevant to the changes
+- [ ] Rebases cleanly onto the latest master
```

Comment:
But still visible to everyone, no? 

I'm ok with leaving it as-is. We can always revisit if anyone else seems to find it a little confusing - it might just be me.